mono/metadata/exception.c

```diff
@@ -931,9 +930,7 @@ mono_get_exception_runtime_wrapped_checked (MonoObject *wrapped_exception, MonoE
 	method = mono_class_get_method_from_name (klass, ".ctor", 1);
 	g_assert (method);
 
-	params [0] = wrapped_exception;
-
-	mono_runtime_invoke_checked (method, o, params, error);
+	mono_runtime_invoke_checked (method, o, (void**)&wrapped_exception, error);
```

Comment:
Actually, this should be a `(gpointer*)` instead of `(void**)`

Reply:
Prototype of mono_runtime_invoke_checked is void**. Change it too?

Reply:
Let's only change what we are changing here. We can change the signature if we come around to changing it for a better reason.

Reply:
Also in many cases the array size and number of parameters can be more coupled by initializing the array with the parameters, i.e. gpointer args [ ] = { foo, bar };